add contents to doc/wise_agents_architecture.md i fixes #148

maeste opened the pull request.

fjuma commented on docs/index.md:
```diff
@@ -1,7 +1,19 @@
+
+Wise agents is an implementation of a [multi agents framework](./multi_agents.md) in Python.
+
+Wise agents is designed as a fully distributed multi agents framework implementation, where agents and their model can be deployed in a cloud architecture. Agents communicate each other using different distributed mechanism and are designed with enterprise needs in mind in terms of security, reliability and privacy.
```
s/model/models

fjuma replied:
s/communicate each other/communicate with each other

fjuma commented on docs/wise_agents_architecture.md:
```diff
@@ -1,28 +1,42 @@
+The following image provides a high-level overview of the architecture and flow:
 
-#### Ollama for local models
+![architecture](/images/architecture.jpg)
```
Just adding a question about the architecture diagram here since I can't comment on it below. Just wanted to check what's meant by "Assistant Agent" vs. "Agent" in the diagram below? Is the distinction that an Assistant Agent can make use of tools?

maeste replied:
It's more an agent able to interact with the user

fjuma reviewed (dismissed):
I've added some very minor comments so adding my approval in case we want to get this merged and address the small comments after.

maeste commented:
> I've added some very minor comments so adding my approval in case we want to get this merged and address the small comments after.

I've fixed what suggested and add an answer about Assitant Agent. Please have a look and if it's ok we can merge it

fjuma commented:
@maeste Thanks, I've published the docs on the website.
